pkg/appgw/internaltypes_test.go

```diff
@@ -12,7 +12,7 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-// appgw_suite_test.go launches these Ginkgo test
```

Comment:
I see that `internaltypes.go` is also being listed as changed files although it has no changes. Can you see what is changing it ?

Reply:
I believe this is an artifact of my setup. Corrected w/ `git update-index --chmod=-x pkg/appgw/internaltypes.go`